src/components/CountdownPane/CountdownPane.js

```diff
@@ -0,0 +1,32 @@
+import "./CountdownPane.css"
+import sunface from "../../assets/sunface.png"
+
+export const CountdownPane = () => {
+  return (
```

Comment:
Will you need countdown logic from the backend, or are you able to make a countdown in reference to the future date we're sending you?

Reply:
I think we can do this on either end! It won't be too hard for us to do it on the front